src/components/IOUConfirmationList.js

```diff
@@ -210,6 +187,7 @@ class IOUConfirmationList extends Component {
 
             const formattedSelectedParticipants = this.getParticipantsWithAmount(selectedParticipants);
             const formattedUnselectedParticipants = this.getParticipantsWithoutAmount(unselectedParticipants);
+            const formattedParticipants = _.union(formattedSelectedParticipants, formattedUnselectedParticipants);
```

Comment:
Was there an issue when just using the `formattedSelectedParticipants` participants? What is the `union` for?

Reply:
It wasn't really a problem, just a quirky implementation where we had:

1. A section titled `Who was there` with all the selected participants that would always show
1. A section without a title with all the unselected participants that would only show if there were any unselected participants.

The effect for the user appeared to be a single section titled `Who was there` with all the selected participants followed by any unselected participants. That's exactly what we're doing here – makes the code a bit more in-line with what the user sees.

Overall this change probably wasn't necessary.

Reply:
Nice, so we can revert it?